lib/uplinkc/uplink.go

```diff
@@ -27,6 +27,7 @@ func new_uplink(cfg C.UplinkConfig, cerr **C.char) C.UplinkRef {
 	scope := rootScope("") // TODO: pass in as argument
 
 	libcfg := &uplink.Config{} // TODO: figure out a better name
+	// TODO: add a way to support logging
```

Comment:
Would be good to add a ticket for this. I assume, this will be highly requested once we are in public Beta

Reply:
https://storjlabs.atlassian.net/browse/V3-2302

Reply:
Could you add the ticket number to the `TODO` comment?

Reply:
added